### PR TITLE
Restaurant page

### PR DESCRIPTION
**Pull request by arikarim**
A Restaurant page built with JavaScript that contains the home page, menu page, and contact page.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,8 +12,16 @@
   "author": "",
```
Comment (MahmoudBakr23) [inline package.json:13]: In the scripts. add the following line:-

```"start": "webpack serve"```

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "no-use-before-define": "off",
+    "no-unused-vars": "off"
```
Comment (MahmoudBakr23) [inline .eslintrc.json]: Please remove these two lines.

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
## **Changes Required 🛑♻**

<hr>

Hello there :)
Great work and effort you made into this app.

## **Highlights**

- Good Linter set up ✔
- Great code separation ✔
- Webpack is well-configured ✔
- Also I liked the design. It looks great ✔

However, there is always room for improvement. Please consider my comments below

## **To Improve**

- [x] Please include the steps and instructions on how to run the app locally in your README file.
- [x] Kindly write a more descriptive PR summary.
descriptive_prs

- [x] Please make sure to not modify the original ESlint configuration file. Instead, try to fix the errors manually. Please look at my inline comment below

## **OPTIONAL**

- It'd be very nice if you add a script to open a live server in your package.json file. See my inline comment below.

<hr>

Good luck and happy coding 💻👨‍💻
Please contact me on Slack with the name "Mahmoud Bakr" if you got any questions. 🙏

**Comment by arikarim** [comment]
Hello, thanks very much for the review.

I have done all the changes, kindly have another review.

**Review by oloomoses — APPROVED** [review]
# Congratulations, Project Approved. :muscle: :tada: :clap: 
**Great job working on the suggested changes from the previous reviews.
You have passed all the requirements for this project :muscle: 
Feel free to merge and close this pull request.**

**I wish you the best in your next project and see you at the top :rocket:** 
#### Happy Coding
_Slack: Oloo Moses_

**Pull request merged**